dask/dataframe/tests/test_dataframe.py

```diff
@@ -1821,3 +1822,14 @@ def test_columns_assignment():
     ddf[['a', 'b']] = ddf2[['y', 'z']]
 
     eq(df, ddf)
+
+
+def test_idxmaxmin():
+    pdf = pd.DataFrame(np.random.randn(100, 5), columns=list('abcde'))
```

Comment:
It would be good to expand this test with a few different kinds of indices to calm some of the concerns above.  Perhaps something like the following:

```python
for idx in np.arange(100), sorted(np.random.random(size=100)), ... something with object dtype ...:
    pdf = pd.DataFrame(..., index=idx)
    ...
```